### DISCUSSION TITLE
NVDash full configuration API needed 🥲

### DISCUSSION THREAD

**Discussion by mnkhod** [NvChad/NvChad#1962] [Q&A] [answered]
I been trying to change the background blue of NVDash for hours now.... There is no documentation regarding Nvdash's full configuration :< anybody got any ideas?

![Screenshot 2023-04-26 at 12 50 22](https://user-images.githubusercontent.com/46494779/234472795-a8ec265b-d1e2-4fd7-90e5-cefc8eae13ae.png)

**Comment by siduck** [comment, marked as answer]
hi @mnkhod  please read https://nvchad.com/docs/config/theming 

All you have to do is change colors for nvdash highlight groups!

**Comment by mnkhod** [comment]
omg thank you 🥹 , it was this the whole time shjdhajlfkdf

![image](https://user-images.githubusercontent.com/46494779/234477577-ba9f9bc4-23d1-4aa2-81a8-b03b03ce1b3e.png)
